canonical-web-and-design/jaas-dashboard
# Add Prettier

## Done

Copy over Prettier config from juju-gui

## QA

- Run `./run exec yarn run prettier-check`
- Verify no errors
